### ISSUE TITLE
Google Maps Engine Lite deprecated and deleted

### ISSUE THREAD

**Issue by acrymble** [programminghistorian/jekyll#210]
This affects the Intro to Google Maps and Google Earth lesson: http://programminghistorian.org/lessons/googlemaps-googleearth

The text at the link: https://mapsengine.google.com/about/index.html

>  Thank you for stopping by.

Google Maps Engine has been discontinued. We want to thank all our loyal Google Maps Engine customers.

Sincerely,

The Google Maps Engine team
Frequently-asked questions

```
What will happen to my Google Maps Engine data?

All data stored with Google Maps Engine will be systematically deleted from Google servers.
Will there be any way to retrieve my subscription data from Google in the future?

No -- all data stored in Google Maps Engine will be permanently and irrevocably deleted. Google will not be able to recover any Google Maps Engine data for any user after February 1st, 2016.
```

---

This looks like a fairly fundamental problem to me, but I havn't had a chance to check yet.

**Comment by ianmilligan1** [comment]
My understanding is that Google My Maps replaced Google Maps Engine, and retained quite a bit of the functionality. 

https://www.google.com/maps/d/

Pinging @jburnford.

**Comment by ianmilligan1** [comment]
Fixed with #215
